### ISSUE TITLE
Found a bug (and fixed it) running Makefile on Debian Testing

### ISSUE THREAD

**Issue by cobbdzon** [PRATAP-KUMAR/gdm-extension#3]
### Background
I just updated my Debian from Bookworm (Debian 12) to Trixie (Debian Testing), and it came with GNOME 46. Now I discovered that you aren't able to change the GDM shell and icon theme. This made me search for a solution which landed me here.

### The bug
```
cobbalt@victuskiosk:~/Documents/Misc/gdm-extension$ sudo make install
[sudo] password for cobbalt: 
gnome-extensions install -f "gdm-extension@pratap.fastmail.fm.zip"
localuser:gdm being added to access control list
X Error of failed request:  BadValue (integer parameter out of range for operation)
  Major opcode of failed request:  109 (X_ChangeHosts)
  Value in failed request:  0xd
  Serial number of failed request:  7
  Current serial number in output stream:  9
sudo: unknown user gdm
sudo: error initializing audit plugin sudoers_audit
make: *** [Makefile:34: install] Error 1
```
TLDR, the user 'gdm' is not found

### The solution
Using the `w` command I found that in Debian, the gdm user is instead 'Debian-gdm'

From:
```
@xhost si:localuser:gdm
@sudo -u gdm dbus-launch dconf reset -f /
@sudo -u gdm dbus-launch gsettings set org.gnome.shell enabled-extensions "['$(UUID)']"
@sudo -u gdm dbus-launch gsettings set org.gnome.desktop.peripherals.touchpad tap-to-click true
```

To:
```
@xhost si:localuser:Debian-gdm
@sudo -u Debian-gdm dbus-launch dconf reset -f /
@sudo -u Debian-gdm dbus-launch gsettings set org.gnome.shell enabled-extensions "['$(UUID)']"
@sudo -u Debian-gdm dbus-launch gsettings set org.gnome.desktop.peripherals.touchpad tap-to-click true
```

### Some other issues I had
The README.md is incomplete, imo it should include the packages needed for the Makefile such as `zip` and `dbus-x11` (I use X11, I don't know if it differs in other Linux desktop builds)

**Comment by PRATAP-KUMAR** [comment]
Hi, Thank you for raising this issue.
I will try to look into it in few days.

**Comment by PRATAP-KUMAR** [comment]
Hi @cobbdzon I have edited Makefile to check for 'Debian-gdm' user exists or not and based on that the GDM_USER variable will assign 'Debian-gdm' or 'gdm' and runs the `Makefile`.

I have also included `zip` and `dbus-x11` packages as prerequisites.

Thank You.

**Comment by cobbdzon** [comment]
Thank you for your assistance!
